akka-http/src/main/scala/akka/http/scaladsl/server/PathMatcher.scala

```diff
@@ -321,6 +332,11 @@ trait ImplicitPathMatcherConstruction {
 trait PathMatchers {
   import PathMatcher._
 
+  def not(self: PathMatcher[_]): PathMatcher0 =
```

Comment:
So, we prefer `not("test")` over `"test".not`. Seems to make sense for this one.